data/scripts/set_wallpaper

```diff
@@ -227,17 +227,35 @@ fi
 # For simple WMs, use either feh or nitrogen
 # Implementation note: this uses spaces around list items to enforce matching whole words.
 # This also means that an empty variable won't cause false positives, since it expands to "  "
-SIMPLE_WMS=("bspwm" "dwm" "herbstluftwm" "i3" "i3-gnome" "i3-with-shmlog" "jwm" "jdwm" "LeftWM" "openbox" "qtile" "qtile-venv" "spectrwm" "xmonad")
-if [[ " ${SIMPLE_WMS[*]} " = *" $XDG_CURRENT_DESKTOP "* || " ${SIMPLE_WMS[*]} " = *" $XDG_SESSION_DESKTOP "* ||
-      " ${SIMPLE_WMS[*]} " = *" $DESKTOP_SESSION "* ]]; then
-	if command -v "feh" >/dev/null 2>&1; then
-	      # TODO: This should take the scaling parameter $4 into account and use other options than --bg-fill
-        feh --bg-fill "$WP" 2> /dev/null
+
+SIMPLE_WMS=(
+  "bspwm"
+  "dwm"
+  "herbstluftwm"
+  "i3"
+  "i3-gnome"
+  "i3-with-shmlog"
+  "jwm"
+  "jdwm"
+  "LeftWM"
+  "openbox"
+  "qtile"
+  "qtile-venv"
+  "spectrwm"
+  "xmonad"
+)
+
+for wm in "${SIMPLE_WMS[@]}"; do
+  if [[ "$XDG_CURRENT_DESKTOP $XDG_SESSION_DESKTOP $DESKTOP_SESSION" = *"$wm"* ]]; then
```

Comment:
So this concatenates all of these variables and does a substring check against our simple wm variable.

For example:

`"none+xmonad none+xmonad none+xmonad"` would substring match against `xmonad`